src/components/structures/MatrixChat.tsx

```diff
@@ -2135,6 +2137,17 @@ export default class MatrixChat extends React.PureComponent<IProps, IState> {
             return null;
         }
 
+        const opts: BannerOpts = { banner: undefined };
+        ModuleRunner.instance.invoke(BannerLifecycle.Banner, opts);
+        if (opts.banner) {
+            view = (
+                <div className="mx_MatrixChat_wrapper">
```

Comment:
From an implementation point of view, I don't think we would need to add a wrapper as the banner could be slotted as a sibling of the `MatrixChat` and the parent section `id="matrixchat"` is a flex container that applies the same CSS that you declared for `mx_MatrixChat_wrapper`. 
I'd be advocating for a leaner HTML structure

Reply:
Changed implementation to place the Banner as a sibling of the `mx_MatrixChat`. Had to update existing `mx_MatrixChat_wrapper` with `flex-direction: column` for the Banner to be shown on the top of Element. 

I have created a PR to the module API with a suggestion to add the Banner: https://github.com/matrix-org/matrix-react-sdk-module-api/pull/13